go.mod

```diff
@@ -15,8 +17,17 @@ require (
 	k8s.io/api v0.0.0-20191112020540-7f9008e52f64
 	k8s.io/apimachinery v0.0.0-20191111054156-6eb29fdf75dc
 	k8s.io/autoscaler/vertical-pod-autoscaler v0.0.0-20191115143342-4cf961056038
-	k8s.io/client-go v0.0.0-20191109102209-3c0d1af94be5
+	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/klog v1.0.0
 )
 
+replace (
```

Comment:
I believe we don't need these anymore.

Reply:
Confirmed that we can remove these. Once you add the revision hashes I'd mentioned earlier, run `go mod tidy` and `go mod vendor`, it should work.